ponyup-init.sh

```diff
@@ -3,7 +3,12 @@
 set -o errexit
 set -o nounset
 
-default_prefix="$HOME/.local/share"
+if [ -z "$XDG_DATA_HOME" ]; then
```

Comment:
This should be changed to `-z "${XDG_DATA_HOME+x}"` to handle the case that the environment variable is unset.

Reply:
Fixed.